release/cli/pkg/util/semver/semver.go

```diff
@@ -0,0 +1,89 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package semver
```

Comment:
Yeah we do but the main eks-A module already imports the release module so we can't import that module (which contains the `semver` package) from release code because it would cause a cyclic import. So I just ported over just the parts we need here.

Reply:
Well upon taking another look it seems like the imports have changed after Guillermo's release module refactor and now the release module imports the EKS-A module. So I'll change this to use the pkg/semver from the EKS-A module.